lib/enterpriseSearch.js

```diff
@@ -27,6 +27,23 @@ class EnterpriseSearchClient {
   destroyDocuments(contentSourceKey, documentIds) {
     return this.client.post(`/sources/${contentSourceKey}/documents/bulk_destroy`, documentIds)
   }
+
+  /**
+   * Lists users permissions for a content source identified by the contentSourceKey.
+   */
+  getPermissions(contentSourceKey, pageParams) {
+    const pageParamsForQuery = pageParams
+      ? {
+          ...(pageParams.current && { 'page[current]': pageParams.current }),
```

Comment:
This is not a pattern I've seen very often, but I find it very concise: https://stackoverflow.com/a/40560953

Reply:
I like this pattern a lot.